pkg/hosts/hosts.go

```diff
@@ -154,7 +204,17 @@ func (h *Hosts) Contains(ip, host string) bool {
 		return false
 	}
 
-	return h.File.Has(ip, host)
+	lines := h.File.GetHostsFileLinesByAddress(ip)
```

Comment:
I keep original implementation, just update code with new lib usage, because I can’t find how `Contains` func used.

Maybe we can delete `Contains`?